Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -21,21 +23,22 @@
 #include <LibGfx/ImageFormats/WebPWriter.h>
 #include <LibTest/TestCase.h>
 
-static ErrorOr<NonnullRefPtr<Gfx::Bitmap>> expect_single_frame(Gfx::ImageDecoderPlugin& plugin_decoder)
+static ErrorOr<NonnullRefPtr<Gfx::Bitmap>> expect_single_frame(Gfx::ImageDecoderPlugin& plugin_decoder, bool is_gif)
```

Comment:
`is_gif` would apply to all animated formats, wouldn't it? Do we want to give it a generic name (`animatable_format`?) right away, to accommodate other formats like APNG as well?

Reply:
I don't know APNG enough to predict that. But we definitely have some weird thing that are currently specific to the GIF decoder. As an example, if a frame has a duration of zero, it will replace it by 100, which isn't very practical for testing. I would rather keep it very specific for now and change it later on.